Good-Bids/goodbids
# [#537] Network Admin Pages

# Summary

This PR adds 3 Network Admin Pages: Nonprofits, Auctions, and Bidders.

## Issues

* #537 

## Testing Instructions

1. Visit the Network Admin
2. Click on each page under GoodBids > {Page Name}
3. Click links within each table to ensure the go to the correct place.
4. Sort each column to ensure sorting is working as expected
5. Adjust the Screen Options to change the amount per page to display
6. Click on the Filters at the top to ensure the filters are working as expected

## Screenshots
<img width="1102" alt="Screenshot 2024-02-29 at 3 31 26 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/ec91046a-4f6a-4ce3-bb42-e945d5447803">

<img width="1105" alt="Screenshot 2024-02-29 at 3 32 32 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/2a03d78d-e423-4036-b292-b3dc50a04218">

<img width="1107" alt="Screenshot 2024-02-29 at 3 32 19 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/be1e56ad-7ec7-46c5-93c9-09d110798c83">
